guides/common/modules/proc_enabling-libvirt-plugin.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="enabling-libvirt-plugin"]
+= Enabling libvirt plugin
+
+Install the libvirt plugin to attach a libvirt/KVM compute resource provider to {Project}.
```

Comment:
If you want to get really technical, it's not a plugin. Instead, it's built into Foreman itself where there is some code in place to disable code paths. That's the the installer command is `--enable-foreman-compute-...` and not `--enable-foreman-plugin-...`

Would this be a better phrasing? Feels very redundant. Especially if the procedure below also has effectively the same text.
```suggestion
= Enabling libvirt compute

Install the libvirt compute resource provider to attach libvirt/KVM compute resources to {Project}.
```